amti/clis/create.py

```diff
@@ -64,16 +73,22 @@ def create_batch(definition_dir, data_path, save_dir, live):
 
     client = utils.mturk.get_mturk_client(env)
 
-    batch_dir = actions.create.create_batch(
+    verify_cost_before_upload = verify or (live and (not force))
+
+    batch_dir, upload_flag = actions.create.create_batch(
         client=client,
         definition_dir=definition_dir,
         data_path=data_path,
-        save_dir=save_dir)
+        save_dir=save_dir,
+        verify_cost_before_upload=verify_cost_before_upload)
+
+    preview_message = f'Preview HITs: {worker_url}' if upload_flag \
```

Comment:
Adding the cost to this log message, regardless of whether we prompt for verification, would give a chance for users who accidentally over spent to quickly expire the batch and recover some of their costs.